scripts/generator.sh

```diff
@@ -0,0 +1,46 @@
+APP_NAME="$(echo $1)"
+ENV_NAME="$(echo $2)"
+ENV_NAME_COLOR=$(echo $3)
+if [ $ENV_NAME_COLOR == "blue" ] || [ $ENV_NAME_COLOR == "green" ]; then
+    ENV_NAME_COLOR=$(echo $2-$3)
+fi
+export CONSUL_URL="$(echo $4)"
+export CONSUL_HTTP_ADDR=https://$CONSUL_URL
+export CONSUL_HTTP_TOKEN="$(echo $5)"
+
+wget -qO- https://github.com/cycloidio/inframap/releases/download/v0.6.7/inframap-linux-amd64.tar.gz | tar xvz
```

Comment:
better to have it in our ECR (main account) or in S3 bucket (also in main) with bucket policy to allow access only from our AWS accounts (organization)